src/plugins/intel_cpu/src/nodes/common/tile_broadcast_utils.cpp

```diff
@@ -250,7 +251,10 @@ void TileBroadcastCommon::optimizedExecute(const MemoryPtr& srcMemory, const Mem
     auto srcData = reinterpret_cast<const char *>(srcMemory->GetPtr());
     auto dstData = reinterpret_cast<char *>(dstMemory->GetPtr());
 
-    if (optimizedParams.srcStrides[5] == 0) {
+    if (srcMemory->getStaticDims() == dstMemory->getStaticDims()) {
+        const auto prc = dstMemory->getDesc().getPrecision();
+        cpu_convert(srcData, dstData, prc, prc, optimizedParams.copySize / prc.size());
```

Comment:
We need to put the comment why cpu_convert is used and not cpu_memcpy. I would also suggest to add TODO: label that will describes our desire to move fast copy logic inside the cpu_memcpu and reuse it everywhere